### DISCUSSION TITLE
mongoose.exe windows10 .conf problem

### DISCUSSION THREAD

**Discussion by PioKrk** [cesanta/mongoose#2888] [Ask questions, get answers] [answered]
I am configuring the mongoose.exe server with mongoose.conf file, which looks like this:
document_root .\www
listening_ports 1234

After starting mongoose.exe I get into CMD:
18ba2891 2 main.c:96:                   Mongoose version : v7.12
18ba28a1 2 main.c:97:                   Listening on     : http://0.0.0.0:8000
18ba28b0 2 main.c:98:                   Web root         : [C:\XXX\mongoose]

So the mongoose.conf file is not read by the mongoose.exe.
But when I start from CMD like: mongoose.exe -l http://0.0.0.0:1234 -d .\www

Microsoft Windows [Version 10.0.19045.4780]
(c) Microsoft Corporation. All rights reserved.

C:\Users\XXX\mongoose>mongoose.exe -l http://0.0.0.0:1234 -d .\www
18bc1260 2 main.c:96:                   Mongoose version : v7.12
18bc126f 2 main.c:97:                   Listening on     : http://0.0.0.0:1234
18bc127f 2 main.c:98:                   Web root         : [C:\Users\XXX\mongoose\www]

This looks OK. 
I tried to follow this description: https://github.com/openalpr/mongoose-mit/blob/master/UserManual.md
but it seems that I did something wrong or in this version of mongoose.exe configuration looks different. 
Please tell me how to properly configure the mongoose.exe server on win10?

**Comment by scaprile** [comment, marked as answer]
We don't release no mongoose.exe. Wherever you got that binary, it is a convenience or a hack. In the first case, it may correspond to some old version of https://github.com/cesanta/mongoose/tree/master/tutorials/http/http-server. In the second case, we don't know.
Please don't ask us about instructions you got from somewhere else.

**Reply by PioKrk** [reply]
Sorry, but there is still an available exe version on this link:
https://mongoose.ws/binary/#

**Reply by scaprile** [reply]
> it is a convenience or a hack. In the first case, it may correspond to some old version of https://github.com/cesanta/mongoose/tree/master/tutorials/http/http-server.
